package.json

```diff
@@ -65,8 +65,8 @@
         "jest-cli": "^19.0.2",
         "prettier": "1.10.2",
         "raw-loader": "^0.5.1",
-        "react": "^16.2.0",
-        "react-dom": "^16.2.0",
+        "react": "^16.3.0",
+        "react-dom": "^16.3.0",
```

Comment:
Still unsure if we should keep the previous one for now and just state that <Table> needs 16.3.0, thoughts?

Reply:
Well this is for the storybook React so it must me 16.3 for the Table to work as is, but the peerDependency might need to be updated depending on what we want to do...

Reply:
Ah that's true, my bad.